src/dotnet-core-uninstall/Shared/Configs/CommandLineConfigs.cs

```diff
@@ -133,6 +133,10 @@ internal static class CommandLineConfigs
             RuntimeInfo.RunningOnWindows ? LocalizableStrings.ForceOptionDescriptionWindows
             : LocalizableStrings.ForceOptionDescriptionMac);
 
+        public static readonly Option IgnoreUpperLimitOption = new Option(
+            "--ignore-upper-limit",
```

Comment:
I'm not sure if it makes more sense to add this new `--ignore-upper-limit` param, or if we should just extend `--force` to allow uninstallation of versions larger than the declared upper limit.

Reply:
@joeloff @baronfel I was looking at improving preview uninstall support for future versions. My initial thought was to add this `--ignore-upper-limit` parameter, but after thinking a bit more I think the UX may be better if we overload `--force` to also handle this case instead?